sematic/scheduling/kubernetes.py

```diff
@@ -349,16 +349,23 @@ def schedule_resolution_job(
     resolution_id: str,
     image: str,
     user_settings: Dict[str, str],
+    max_parallelism: Optional[int] = None,
 ) -> ExternalJob:
+
     namespace = get_user_settings(SettingsVar.KUBERNETES_NAMESPACE)
     external_job = KubernetesExternalJob.new(
         try_number=0,
         run_id=resolution_id,
         namespace=namespace,
         job_type=JobType.driver,
     )
+
     logger.info("Scheduling job %s", external_job.kubernetes_job_name)
     args = ["--run_id", resolution_id, "--resolve"]
+
+    if max_parallelism is not None:
+        args += ["--max_parallelism", str(max_parallelism)]
```

Comment:
For rerun, I used a dash to separate words: `--rerun-from`. I find it more esthetic and also is more GNU-like: https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html#:~:text=with%20hyphens%20to%20separate%20words

Reply:
Above, `--run_id` is spelled with an underline. We should keep things consistent, either way.

Reply:
Ah yeah. I still prefer `-` 😬

Reply:
Done.